ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/Components/IntegrationHelpIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ReactElement } from 'react';
+import { Popover } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+
+export type IntegrationHelpIconProps = {
+    helpTitle: string;
+    helpText: React.ReactElement;
+    ariaLabel: string;
+};
+
+function IntegrationHelpIcon({
+    helpTitle,
+    helpText,
+    ariaLabel,
+}: IntegrationHelpIconProps): ReactElement {
+    return (
+        <Popover headerContent={helpTitle} bodyContent={helpText}>
+            <button
+                type="button"
+                aria-label={ariaLabel}
+                onClick={(e) => e.preventDefault()}
+                aria-describedby="simple-form-name-01"
```

Comment:
We can remove this. `simple-form-name-01` is hardcoded as the `aria-describedby` value, which isn't useful.